extensions/data-plane/data-plane-iam/src/main/java/org/eclipse/edc/connector/dataplane/iam/service/DataPlaneAuthorizationServiceImpl.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *  Copyright (c) 2024 Contributors to the Eclipse Foundation
```

Comment:
we don't have this copyright anywhere else, maybe it's better to leave the old one in place?

Reply:
yes, that was an intellij inititative :D